test/Microsoft.TestPlatform.AcceptanceTests/AcceptanceTestBase.cs

```diff
@@ -45,12 +45,20 @@ public class AcceptanceTestBase : IntegrationTestBase
     public const string DesktopRunnerTargetRuntime = "win7-x64";
     public const string CoreRunnerTargetRuntime = "";
     public const string InIsolation = "/InIsolation";
-
+    
     public const string NETFX452_48 = "net452;net461;net472;net48";
     public const string NETFX451_48 = "net452;net461;net472;net48";
     public const string NETCORE21_50 = "netcoreapp2.1;netcoreapp3.1;net5.0";
     public const string NETFX452_NET50 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1;net5.0";
     public const string NETFX452_NET31 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1";
+    public const string DEFAULT_RUNNER_NETFX = "net451";
+    /// <summary>
+    /// Our current defaults for .NET and .NET Framework.
+    /// </summary>
+    public const string DEFAULT_RUNNER_NETFX_AND_NET = $"{DEFAULT_RUNNER_NETFX};netcoreapp2.1"; 
+    public const string DEFAULT_HOST_NETFX_AND_NET = "net451;netcoreapp2.1";
+    public const string LATEST_LEGACY = "Latest;LatestStable;LatestPreview;MostDownloaded;PreviousStable;LegacyStable";
+    public const string LATESTSTABLE_LEGACY = "LatestStable;LatestPreview;MostDownloaded;PreviousStable;LegacyStable";
```

Comment:
I will need to work on the naming here, but the goal is to have a default for runner and host (and collector?), that we will use everywhere, so we can move easily into next version of tfms without having to change it everywhere. But still have the possibility to specify custom bigger or smaller range of frameworks for testing.

And same for the versions (e.g. Latest), I did not try to apply the idea to anything other than test framework version, yet, but I don't want to have concrete versions in our code, so we can easily move to a newer version every month / release.